JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -6,7 +6,34 @@
 
 import Foundation
 
-// 과일 저장소 타입
-class FruitStore {
+
+final class FruitStore {
+    
+    // MARK: - Properties
+    
+    private var fruitStock: [Fruit: Int] = [
+        .strawberry : 10,
+        .banana: 10,
+        .pineapple: 10,
+        .kiwi: 10,
+        .mango: 10] {
+            didSet {
+                NotificationCenter.default.post(
+                    name: Notification.Name("stcokDidChanged"),
+                    object: self
+                )
+            }
+        }
+    
+    // MARK: - Methods
+    
+    func consume(fruit: Fruit, count: Int) {
+        guard let remainCount = fruitStock[fruit] else { return }
+        fruitStock.updateValue(remainCount - count, forKey: fruit)
+    }
     
+    func quantity(of: Fruit) -> Int {
```

Comment:
```suggestion
    func quantity(of fruit: Fruit) -> Int {
```
=> 이건 너무 대충인것 같아요.. of 로 변수명을 지으라는게 아니라 인자 레이블로서 of 를 쓰라는 말씀이었습니다 🥲
인자를 `of fruit`으로 쓰면 밖에서는 `(of: fruit)`, 안에서는 fruit를 사용하게 되어 가독성이 좋습니다.

Reply:
정확한 사용법을 숙지하고 사용하도록 하겠습니다!

Reply:
넵 참고로 대충이란 말은 강렬하게 기억되시라고 표현한거니 신경 안쓰셔도 됩니다 ㅎㅎ
Swift는 어규먼트 레이블이라는게 있다는 것을 이번 기회에 제대로 공부하시면 좋겠네요! 호호 👍👍👍